scripts/commands/convertApiDocsToHistorical.ts

```diff
@@ -125,12 +125,12 @@ async function generateJsonFiles(
   // by any subfolder starting with a number (historical version folders)
   // or a release note file
   const linksToUptade = new RegExp(
-    '"url": "/api/' + pkgName + "/(?!release-notes)(?![0-9])",
+    '"url": "/api/' + pkgName + '(?=/|")(?!/release-notes)(?!/[0-9])',
```

Comment:
What's up with the `"` in the regex? Would it work to instead do `/?(?!relese-notes)(?![0-9]`? Don't use the positive lookahead for `/|"`